### PR TITLE
Pivotal ID # 172476764: Refresh Interdatabase Support

### PR DESCRIPTION
**Pull request by jhoanmanuelms**
https://www.pivotaltracker.com/story/show/172435555

Endpoint to get the extended submission

### CODE REVIEW

File: commons/commons-model-extended/build.gradle.kts [hunk 1]
```diff
@@ -8,7 +10,8 @@ dependencies {
 
     implementation(Arrow)
     implementation(Guava)
-    implementation(kotlin("stdlib"))
+    implementation(KotlinStdLib)
+    implementation(JacksonKotlin)
```
Comment (Juan-EBI) [inline commons/commons-model-extended/build.gradle.kts:14]: I think this one is not needed

Comment (Juan-EBI) [inline commons/commons-model-extended/build.gradle.kts:14]: I think this one is not needed

File: submission/submission-webapp/src/main/resources/application.yml [hunk 2]
```diff
@@ -23,10 +23,10 @@ spring:
       max-request-size: 20GB
       location: ${app.tempDirPath}
 
-logging.file: logs/application_logs.txt
 logging:
   file:
     max-history: 7
+    name: logs/application_logs.txt
```
Comment (Juan-EBI) [inline submission/submission-webapp/src/main/resources/application.yml:29]: did you test this?

Comment (jhoanmanuelms) [reply submission/submission-webapp/src/main/resources/application.yml:29]: Yes, it was deployed in dev and the logs were properly generated.

**Pull request merged**